docs/docs/FAQ.mdx

```diff
@@ -360,6 +360,33 @@ cap_drop:
 
 </details>
 
+### How can I monitor the health of the Immich containers?
+
+All of the containers have health monitoring scripts that are disabled by default. To enable them, add the following to your `docker-compose.yml`.
+
+```diff
+services:
+  immich-server:
+    ...
++   healthcheck:
++     test: npm run healthcheck
+
+  immich-machine-learning:
+    ...
++   healthcheck:
++     test: python3 healthcheck.py
+
+  redis:
+    ...
++   healthcheck:
++     test: redis-cli ping
+
+  database:
+    ...
++   healthcheck:
++     test: ["CMD-SHELL", "pg_isready", "--dbname=${DB_DATABASE_NAME}"]
```

Comment:
Out of curiosity, does it perform the database checksums as you added in #9391?

Reply:
Good idea! I just added this.